### ISSUE TITLE
Install problem?

### ISSUE THREAD

**Issue by pefs44** [gpilab/framework#1]
Hi GPI, 

OSx 10.9.5 
whether I install from the dmg or the script, in the end when I try to run GPI I get the "No Clinical Use ..." message, a short wait and then... 
/Users/paul/gpi_stack/bin/gpi: line 44: 32143 Abort trap: 6           $GPI_LINK $GPI_LAUNCH $@
open to suggestions about what might be missing.

apologies if I've dropped this on the wrong forum.

Paul Summers

**Comment by aganders3** [comment]
Hi @pefs44,

Can you please post the full console output you get by running GPI using `gpi --log=debug`?

An easy way to do this is using a paste service e.g. sprunge.us. Run this command in your terminal:
`% gpi --log=debug | curl -F 'sprunge=<-' http://sprunge.us`

It should give you a link (e.g. http://sprunge.us/bJfF, this is the output when I start GPI with debug logging) after the crash, which you can post here.

**Comment by pefs44** [comment]
here you go:   http://sprunge.us/dWOR

**Comment by aganders3** [comment]
Hm, that link is blank. Maybe you ran the command with the `%` in front? I meant that as a placeholder for your command prompt. Sorry - can you try again with just: `gpi --log=debug | curl -F 'sprunge=<-' http://sprunge.us`?

**Comment by pefs44** [comment]
no %, but you're right - nothing in sprunge:
pauls-MacBook-Pro:~ paul$ gpi --log=debug | curl -F 'sprunge=<-' http://sprunge.us
/Users/paul/gpi_stack/bin/gpi: line 44: 35787 Abort trap: 6           $GPI_LINK $GPI_LAUNCH $@
http://sprunge.us/SNYW
pauls-MacBook-Pro:~ paul$

without sprunge: 
pauls-MacBook-Pro:~ paul$ gpi --log=debug
GPI 1.0.0 (2015-11-13)  Copyright (C) 2014 Dignity Health
This program comes with ABSOLUTELY NO WARRANTY; see the LICENSE for details.

    NO CLINICAL USE.  THE SOFTWARE IS NOT INTENDED FOR COMMERCIAL PURPOSES
AND SHOULD BE USED ONLY FOR NON-COMMERCIAL RESEARCH PURPOSES.  THE SOFTWARE MAY NOT IN ANY EVENT BE USED FOR ANY CLINICAL OR DIAGNOSTIC PURPOSES.  YOU ACKNOWLEDGE AND AGREE THAT THE SOFTWARE IS NOT INTENDED FOR USE IN ANY HIGH RISK OR STRICT LIABILITY ACTIVITY, INCLUDING BUT NOT LIMITED TO LIFE SUPPORT OR EMERGENCY MEDICAL OPERATIONS OR USES.  LICENSOR MAKES NO WARRANTY AND HAS NOR LIABILITY ARISING FROM ANY USE OF THE SOFTWARE IN ANY HIGH RISK OR STRICT LIABILITY ACTIVITIES.

Tue May  3 21:26:13 2016 - gpi.loader:119 - INFO - SUCCESS: /Users/paul/gpi_stack/lib/python3.5/site-packages/gpi/node-libs/core/display/GPI/Audio_GPI.py
Tue May  3 21:26:13 2016 - gpi.loader:119 - INFO - SUCCESS: /Users/paul/gpi_stack/lib/python3.5/site-packages/gpi/node-libs/core/display/GPI/CrossSection_GPI.py
/Users/paul/gpi_stack/bin/gpi: line 44: 35823 Abort trap: 6           $GPI_LINK $GPI_LAUNCH $@
pauls-MacBook-Pro:~ paul$ 

Paul

**Comment by aganders3** [comment]
OK, thanks...we haven't run into this issue before.

@nick, any ideas?

@pefs44, please also try launching GPI (more) directly with the following command and posting any output:
`/Users/paul/gpi_stack/bin/python /Users/paul/gpi_stack/bin/gpi_launch`

**Comment by pefs44** [comment]
hi, 
as with the simple gpi command, the gpi_launch command gives me the disclaimers, and I see a python "icon" enter into the sidebar, then the icon disappears and the whole lot crashes. The only difference is that with gpi the epitaph is: 
/Applications/GPI.app/Contents/Resources/miniconda/bin/gpi: line 44: 39528 Abort trap: 6           $GPI_LINK $GPI_LAUNCH $@
while gpi_launch gives only
Abort trap: 6 

again adding the pipe to sprunge results in an empty webpage

but, hmmm if I do,
sudo gpi 
I get "Network Canvas" and a gpi menubar. 

let me poke around as to why...

Paul

**Comment by pefs44** [comment]
Well, I can't explain why, but GPI seems to be living. :-)

seeing that sudo gpi worked and for me gpi didn't, I thought maybe a permission out of place, but both the GPI app and script installs were done by me without recourse to root. ls -l shows me as owner of all the files and folders under GPI.app and under gpi_stack though they differ in that for one my group is admin but staff for the other. 

checking on path differences I found that as root, python3.5 was that in /Application/GPI.app while the script version pointed to python in /home/paul/gpi_stack

moved GPI.app to myGPI.app
both my my and the distal point to python in /home/paul/gpi_stack
and both worked, starting with a new terminal each time

moved /Applications/myGPI.app back to GPI.app
both now point to python in /Applications/GPI.app
AND BOTH WORK

I've since rebooted, and still GPI is starting from both the app and script version.
So things seem resolved without knowing why... 

now to figure out what to do with GPI.  

Paul
